backend/internal/model/user.go

```diff
@@ -19,6 +19,7 @@ type User struct {
 	IsAdmin   bool   `sortable:"true"`
 	Locale    *string
 	LdapID    *string
+	Disabled  bool `sortable:"true" gorm:"default:false"`
```

Comment:
```suggestion
	Disabled  bool `sortable:"true"`
```

This has no effect as we are not using Gorm for migrations